addons/website_form/controllers/main.py

```diff
@@ -36,6 +36,10 @@ def website_form(self, model_name, **kwargs):
             id_record = self.insert_record(request, model_record, data['record'], data['custom'], data.get('meta'))
             if id_record:
                 self.insert_attachment(model_record, id_record, data['attachments'])
+                # in case of an email, we want to send it immediately instead of waiting
+                # for the email queue to process
+                if model_name == 'mail.mail':
+                    request.env[model_name].sudo().browse(id_record).send()
```

Comment:
insert_record could return the actual record instead of it's id now so no browse is needed

Reply:
@fja-odoo We're in stable here, so no ;)

Reply:
@qsm-odoo @fja-odoo  I've created a master branch https://github.com/odoo/odoo/pull/42390